src/internal/collection/watch_test.go

```diff
@@ -53,7 +54,7 @@ func newTestEvent(t *testing.T, ev *watch.Event) TestEvent {
 }
 
 type WatchTester interface {
-	Write(id string, value ...string)     // writes a row with the given ID and value
+	Write(item *col.TestItem)             // writes the given row
```

Comment:
refactored some of the test utility functions because they encouraged repetition of magic values